src/commands/transact.ts

```diff
@@ -26,7 +25,12 @@ async function register() {
             return;
         }
 
-        const result: GetAbiResult = await api.rpc.get_abi(contract).catch((err) => {
+        const ultraApi = await Service.api.getUltraApi().catch((err) => {
+            console.log(err);
+            return undefined;
+        });
+
+        const result = await ultraApi.getAbi(contract).catch((err) => {
```

Comment:
`getAbi` is not a function available in `UltraAPI`. I think instead it's `ultraApi.chain.getAbi`:

https://github.com/ultraio/toolkit-template/blob/6109afff71137ed8b670885915984dfc35bd66d5/src/utilities/blockchain.ts#L249